### ISSUE TITLE
Pretrained models

### ISSUE THREAD

**Issue by loretoparisi** [microsoft/NUWA#2]
Thanks for this project, it's simply amazing.
Any plans to share the pre-trained model(s)? That would be super helpful to compare it against CLIP, DALL-E, VQGAN, and these models ensemble combinations.

Thanks a lot

**Comment by chris-aeviator** [comment]
also Code :eyes:

**Comment by johnpaulbin** [comment]
@loretoparisi They already did some small comparisons with the original OpenAI Dall-E model in the paper:

![image](https://user-images.githubusercontent.com/43943237/143473658-6ff42905-714a-47bd-bd31-fd5a6da2ac5c.png)

With clip and VQGan, those specifically aren't designed to generate images, but I'm sure you've seen a google colab somewhere that combines those. I recommend instead checking out Guided-Diffusion and CLIP, it generates much more fine details than CLIP + VQGan could ever do.

**Comment by johnpaulbin** [comment]
but yes, their pre-trained models would be nice to have.

**Comment by loretoparisi** [comment]
> @loretoparisi They already did some small comparisons with the original OpenAI Dall-E model in the paper:
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/43943237/143473658-6ff42905-714a-47bd-bd31-fd5a6da2ac5c.png)
> 
> 
> 
> With clip and VQGan, those specifically aren't designed to generate images, but I'm sure you've seen a google colab somewhere that combines those. I recommend instead checking out Guided-Diffusion and CLIP, it generates much more fine details than CLIP + VQGan could ever do.

Thank you!
For CLIP + Guided Diffusion I'm using the 512x512 patch with unconditional ImageNet diffusion [here](https://colab.research.google.com/drive/1mpkrhOjoyzPeSWy2r7T8EYRaU7amYOOi). 

Didn't test it intensively actually on free Colab with K80 is pretty much slow, at least a V100 is needed [to have](https://twitter.com/rivershavewings/status/1461379472339308547?s=21) a reasonable 10 minutes computing time...

Yes having a Colab with pretrained is the omly way this models cam be compared by ml practitioners.

**Comment by chenfei-wu** [comment]
The model and code will be made public in the near future. But recently we need time to clean up the code to make it simple and easy to reproduce. In addition, we also need to follow the company's public process. Thank you all for your understanding😊.

**Comment by loboere** [comment]
please add colab

**Comment by DexiangHong** [comment]
> The model and code will be made public in the near future. But recently we need time to clean up the code to make it simple and easy to reproduce. In addition, we also need to follow the company's public process. Thank you all for your understanding😊.

Looking forward to the code!

**Comment by Sierkinhane** [comment]
Looking forward to the code TOO !😊

**Comment by josephrocca** [comment]
For the sake of those subscribed to this thread for updates from the authors, it would be nice if there were fewer +1/"me too" type comments. Everyone is clearly excited for the code and pretrained models! :)

**Comment by dcsrgh** [comment]
What a magical work. Waiting the code.

**Comment by You-SB** [comment]
666666

**Comment by YashVardhan-AI** [comment]
i want codeeeeee

**Comment by myname1111** [comment]
When can we expect the code to be released in the worst case scenario?

**Comment by YashVardhan-AI** [comment]
i would like the code as soon as possible i honestly wanna see how good their claims are

**Comment by Sierkinhane** [comment]
Two weeks passed...

**Comment by Sierkinhane** [comment]
Three weeks passed...

**Comment by afiaka87** [comment]
https://github.com/lucidrains/nuwa-pytorch

**Comment by YashVardhan-AI** [comment]
> https://github.com/lucidrains/nuwa-pytorch

yes lol we get another guy(he is amazing btw) release the code before the official code it released nice

**Comment by VictorSanh** [comment]
+1 and bump: would love to see public models and code of NUWA-infinity! on the hugging face model hub (https://huggingface.co/) for instance :)

**Comment by loretoparisi** [comment]
+1 for pre-trained models on HuggingFace

**Comment by Maxlinn** [comment]
still waiting, any progress?

**Comment by myname1111** [comment]
> still waiting, any progress?

Just. Give. Up
Text to image: https://github.com/CompVis/stable-diffusion
Text to video: https://github.com/THUDM/CogVideo

**Comment by Maxlinn** [comment]
> > still waiting, any progress?
> 
> Just. Give. Up Text to image: https://github.com/CompVis/stable-diffusion Text to video: https://github.com/THUDM/CogVideo

those repos seem interesting, thank you!
